cpp/src/arrow/flight/transport/ucx/CMakeLists.txt

```diff
@@ -18,19 +18,15 @@
 add_custom_target(arrow_flight_transport_ucx)
 arrow_install_all_headers("arrow/flight/transport/ucx")
 
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(UCX REQUIRED IMPORTED_TARGET ucx)
-
 set(ARROW_FLIGHT_TRANSPORT_UCX_SRCS
     ucx_client.cc
     ucx_server.cc
     ucx.cc
     ucx_internal.cc
     util_internal.cc)
-set(ARROW_FLIGHT_TRANSPORT_UCX_LINK_LIBS)
 
-include_directories(SYSTEM ${UCX_INCLUDE_DIRS})
-list(APPEND ARROW_FLIGHT_TRANSPORT_UCX_LINK_LIBS PkgConfig::UCX)
+get_target_property(UCX_INCLUDE_DIR ucx::ucx INTERFACE_INCLUDE_DIRECTORIES)
+include_directories(SYSTEM ${UCX_INCLUDE_DIR})
```

Comment:
Any better way than `include_directories`?

Reply:
We can remove this once https://github.com/apache/arrow/pull/12861 is completed. :-)